### ISSUE TITLE
[feature] Create a custom Hummingbot branch that allows trading without needing to start strategy

### ISSUE THREAD

**Issue by martinkou** [pkweitai/hummingbotAI#11]
## Why

The current version of open source Hummingbot requires a strategy to be created first, and all trading actions would be carried out by the strategy. This doesn't allow the kind of retail user trading use case described in the [Hummingbot AI proposal](https://www.notion.so/Hummingbot-AI-Proposal-004f399b6f8e43f1a7948253c02fd8d5).

To fix this, we should create a custom Hummingbot branch that allows for trading and status commands via MQTT, without needing importing a strategy.

## What

* Draft a list of new MQTT commands that Hummingbot AI will need to call, for enabling manual trading from Telegram interface
* Implement the MQTT commands on the custom Hummingbot branch

**Comment by martinkou** [comment]
Created a fork on my side for this: https://github.com/martinkou/hummingbot

**Comment by martinkou** [comment]
I've created a feature branch from the open source Hummingbot at https://github.com/slash-bin/hummingbot/commits/feat/user_directed_trades for this purpose. There'll be two major components to this:

1. The MQTT command controller logic, with the message schema;
2. A `UserDirectedTrade` strategy, which automatically uses all available exchange connectors and interfaces with the MQTT controller.

For demo purpose, we'll package this into a Docker container that auto-creates and auto-launches a `UserDirectedTrade` strategy config. So once the user launches the demo Hummingbot, it'll be ready to accept user directed trading commands from MQTT.

I've started defining the MQTT messages and command controller logic last weekend. I'll be implementing the `UserDirectedTrade` strategy this week.
